xlators/mgmt/glusterd/src/glusterd-locks.c

```diff
@@ -643,8 +649,10 @@ gd_mgmt_v3_unlock_timer_cbk(void *data)
     glusterd_mgmt_v3_lock_timer *mgmt_lock_timer = NULL;
     char *key = NULL;
     int keylen;
+#ifdef DEBUG
     char bt_key[PATH_MAX] = "";
     int bt_key_len = 0;
+#endif
```

Comment:
This can also be moved to the following `#ifdef DEBUG` block.

Reply:
Ack